Tests/BSONTests/BSONObjectIDTests.swift

```diff
@@ -62,8 +55,8 @@ final class BSONObjectIDTests: BSONTestCase {
     }
 
     func testTimestampCreation() throws {
```

Comment:
Running this test locally, it takes 0.001 seconds to run, so I think the risk of us rolling over to the next second should be pretty low. To ensure complete testing, lets add the seconds back in here. If it fails too often we can just revert it.

Reply:
done